language/de-DE/de-DE.com_users.ini

```diff
@@ -49,114 +49,118 @@ COM_USERS_LOGIN_REMEMBER_ME="Angemeldet bleiben"
 COM_USERS_REGISTRATION_ACTIVATION_NOTIFY_SEND_MAIL_FAILED="Beim Senden der Aktivierungs-E-Mail ist ein Fehler aufgetreten"
 COM_USERS_REGISTRATION_ACTIVATION_SAVE_FAILED="Fehler beim Speichern der Aktivierungsdaten: %s"
-COM_USERS_REGISTRATION_ADMINACTIVATE_SUCCESS="Das Benutzerkonto wurde aktiviert und der Benutzer wurde informiert!"
+COM_USERS_REGISTRATION_ADMINACTIVATE_SUCCESS="Das Benutzerkonto wurde aktiviert und der Benutzer wurde informiert."
 COM_USERS_REGISTRATION_BIND_FAILED="Fehler beim Einbinden der Registrierungsdaten: %s"
 COM_USERS_REGISTRATION_COMPLETE_ACTIVATE="Das Benutzerkonto wurde erstellt und ein Aktivierungslink wurde an die eingegebene E-Mail-Adresse verschickt. Vor dem ersten Anmelden muss das Benutzerkonto durch Klicken auf den Aktivierungslink bestätigt werden."
 COM_USERS_REGISTRATION_COMPLETE_VERIFY="Das Benutzerkonto wurde erstellt und ein Bestätigungscode wurde an die eingegebene E-Mail-Adresse verschickt. Das Benutzerkonto wird mit einem Klick auf den Verifizierungslink in der E-Mail verifiziert. Vor dem ersten Anmelden muss aber noch ein Administrator das Benutzerkonto aktivieren."
 COM_USERS_REGISTRATION_DEFAULT_LABEL="Benutzerregistrierung"
 COM_USERS_REGISTRATION_SAVE_FAILED="Registrierung fehlgeschlagen: %s"
 COM_USERS_REGISTRATION_SAVE_SUCCESS="Vielen Dank für die Registrierung. Eine Anmeldung mit dem in der Registrierung angegeben Benutzernamen und Passwort ist nun möglich."
-COM_USERS_REGISTRATION_SEND_MAIL_FAILED="Beim Verschicken der Registrierungs-E-Mail ist ein Fehler aufgetreten! Der Admistrator der Seite wurde hierüber informiert."
+COM_USERS_REGISTRATION_SEND_MAIL_FAILED="Beim Verschicken der Registrierungs-E-Mail ist ein Fehler aufgetreten. Der Admistrator der Seite wurde hierüber informiert."
 COM_USERS_REGISTRATION_VERIFY_SUCCESS="Die E-Mail-Adresse wurde verifiziert. Sobald der Administrator das Benutzer-Konto aktiviert hat, wird automatisch eine weitere E-Mail verschickt."
 COM_USERS_REMIND="Erinnerung"
 COM_USERS_REMIND_DEFAULT_LABEL="Bitte die für das Benutzerkonto hinterlegte E-Mail-Adresse eingeben. Der Benutzername wird dann an diese E-Mail-Adresse geschickt."
 COM_USERS_REMIND_EMAIL_LABEL="E-Mail"
-COM_USERS_REMIND_LIMIT_ERROR_N_HOURS="Die maximale Anzahl zum Zurücksetzen des Passworts wurde erreicht! Bitte in %s Stunden erneut versuchen."
-COM_USERS_REMIND_LIMIT_ERROR_N_HOURS_1="Die maximale Anzahl zum Zurücksetzen des Passworts wurde erreicht! Bitte in einer Stunde erneut versuchen."
+COM_USERS_REMIND_LIMIT_ERROR_N_HOURS="Die maximale Anzahl zum Zurücksetzen des Passworts wurde erreicht. Bitte in %s Stunden erneut versuchen."
+COM_USERS_REMIND_LIMIT_ERROR_N_HOURS_1="Die maximale Anzahl zum Zurücksetzen des Passworts wurde erreicht. Bitte in einer Stunde erneut versuchen."
+COM_USERS_REMIND_REQUEST="Wenn die eingegebene E-Mail-Adresse auf dieser Website registriert ist, wird in Kürze eine E-Mail mit einer Erinnerung gesendet."
```

Comment:
```suggestion
COM_USERS_REMIND_REQUEST="Wenn die eingegebene E-Mail-Adresse auf dieser Website registriert ist, erhalten Sie in Kürze eine E-Mail mit dem verknüpften Benutzernamen."
```

Reply:
Please check. I have deliberately tried not to use "Sie".